.travis.yml

```diff
@@ -9,12 +9,18 @@ dist: bionic
 env:
   global:
     - NODE_OPTIONS="--max_old_space_size=4096"
+    - ETH_PRIVATE_KEY=$TEST_ETH_PRIVATE_KEY
 
 install:
   - pip install codecov pytest-cov
   - pip install -e .
   - pip install -e .[dev]
-  - export ETH_PRIVATE_KEY=$TEST_ETH_PRIVATE_KEY
+  - git clone https://${CI_USER_TOKEN}@github.com/skalenetwork/skale-manager.git
```

Comment:
why this moved to `install` stage?

Reply:
I think contract deploying isn't part of stage test. Also travis ui compresses and hides huge npm i and truffle migrate logs

Reply:
doesn't it cause double deployment during the test and build stages?